src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -63,7 +64,18 @@
                                        (map (fn [message]
                                               [:div {} message])
                                          message-list)])}
-                 {:header "Workflow ID" :starting-width 300}]
+                 {:header "Workflow ID" :starting-width 300
+                  :content-renderer
+                  (fn [workflow]
+                   (let [submission-id (:submission-id props) 
+                         bucketName (:bucketName props)
```

Comment:
These first two can be shortened by using destructuring:
```clojure
{:keys [submission-id bucketName]} props
```
Use your own judgement as to whether that concision adds value.